include/deepworks/optimizer.hpp

```diff
@@ -21,5 +21,22 @@ class SGD {
     Parameters& m_params;
 };
 
+class Momentum {
```

Comment:
Does it work better for your CIFAR network ?

Reply:
Maybe rename it to `SGDMomemtum`? Because momentum is used in other optimizers too